default.nix

```diff
@@ -4,7 +4,12 @@ with (import (fetchTarball https://github.com/NixOS/nixpkgs/archive/fb235c98d839
 stdenv.mkDerivation {
   name = "daedalus";
 
-  buildInputs = [git python27 curl electron nodejs-6_x nodePackages.node-gyp nodePackages.node-pre-gyp ];
+  buildInputs = [
+    nix bash binutils coreutils curl
+    git python27 curl electron nodejs-6_x
+    nodePackages.node-gyp nodePackages.node-pre-gyp
+    gnumake
+  ];
 
   src = null;
```

Comment:
one small nit-pick, this file only ever works with `nix-shell`, so shouldnt it be called `shell.nix`?
ive renamed it in one of my PR's

Reply:
+1 - we can migrate that as part of linux installer :)

Reply:
I agree with both comments.